server/src/main/dist/conf/sidecar.yaml

```diff
@@ -173,6 +167,18 @@ driver_parameters:
   contact_points:
     - "127.0.0.1:9042"
     - "127.0.0.2:9042"
+  username: cassandra
+  password: cassandra
```

Comment:
Super nit: `password: password` to keep consistency with the password fields for keystore and truststore?

Reply:
Since by default `cassandra` user is created in Cassandra with `cassandra` password. Better to keep this way